src/settings.js

```diff
@@ -530,6 +530,11 @@ var MIN_WEBGL_VERSION = 1;
 // Specifies the highest WebGL version to target. Pass -s MAX_WEBGL_VERSION=2
 // to enable targeting WebGL 2. If WebGL 2 is enabled, some APIs (EGL, GLUT, SDL)
 // will default to creating a WebGL 2 context if no version is specified.
+// Note that there is no automatic fallback to WebGL1 if WebGL2 is not supported
+// in the browser at runtime, even if you build with both WebGL1 and WebGL2
```

Comment:
"supported in the browser at runtime" is slightly unclear; how about:
```suggestion
// by the user's device, even if you build with both WebGL1 and WebGL2
```